build/install_requirements.sh

```diff
@@ -39,20 +39,24 @@ sudo apt-get install -y libgstreamer1.0-0 gstreamer1.0-plugins-base \
 # install numpy native lib
 sudo apt-get install -y python3-numpy
 sudo apt-get install -y libjpeg-dev zlib1g-dev
-
+sudo apt-get install -y libssl-dev pkg-config libhdf5-103 libhdf5-dev
 # [backend]
 
 # make sure python sees the packages installed via apt-get
 # export PYTHONPATH=$PYTHONPATH:/usr/lib/python3/dist-packages
 # echo "export PYTHONPATH=$PYTHONPATH:/usr/lib/python3/dist-packages" >> $HOME/.bashrc
 
+pip3 --version
+sudo pip3 install -U pip
+
 # Install Raspberry Pi video drivers
-if $(arch | grep -q arm)
+# if $(arch | grep -q arm)
 # there is no RPI firmware in docker images, so we will install on ARM flag
 #if grep -s -q "Raspberry Pi" /sys/firmware/devicetree/base/model; then
+if grep -s -q "Raspberry Pi" /proc/cpuinfo;
```

Comment:
It looks like this check is failing. I noticed you commented out the earlier arm check and replaced by a Raspibery Pi check. However in the CI environment this fails and the whole section with rpi specific deps does not run.

Reply:
I think this is the line that started up the chain of build error. For some reason I changed it to match just RPi. Good catch!